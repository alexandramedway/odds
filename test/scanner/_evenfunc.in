set iseven = (num) -> 
  return if num % 2 == 0 then true else "false"
