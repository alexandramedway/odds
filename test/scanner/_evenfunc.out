DO
ID
ASN
LPAREN
ID
RPAREN
FDELIM
RETURN
IF
ID
MOD
INT_LITERAL
EQ
INT_LITERAL
THEN
BOOL_LITERAL
ELSE
STRING_LITERAL
